src/app/vault/ciphers.component.ts

```diff
@@ -65,6 +68,8 @@ export class CiphersComponent extends BaseCiphersComponent implements OnDestroy
   // Do not use ngOnInit() for anything that requires sync data.
   async load(filter: (cipher: CipherView) => boolean = null, deleted = false) {
     await super.load(filter, deleted);
+    const profile = await this.apiService.getProfile();
```

Comment:
Do we want to do a http post here?

Reply:
As per your suggestion I used tokenService instead, thank you!